spec/factories/responses.rb

```diff
@@ -15,6 +15,7 @@
     type       { AnsibleTowerClient::FactoryHelper.underscore_string(klass.namespace.last) }
     name       { "#{type}-#{id}" }
     url        { "/api/v1/#{klass.endpoint}/#{id}/" }
+    related    { {"survey_spec" => ["blah blah"]} }
```

Comment:
Is it actually an array of strings?  Usually the "related" are links to other collections, right?